iati/resources.py

```diff
@@ -309,13 +310,24 @@ def folder_name_for_version(version=None):
     """
     if version is None:
         return 'version-independent'
-
-    if version in iati.constants.STANDARD_VERSIONS:
-        return version.replace('.', '-')
-    elif version in [str(major_version) for major_version in iati.constants.STANDARD_VERSIONS_MAJOR]:
-        return version
-    else:
-        raise ValueError("Version {} is not a valid version of the IATI Standard.".format(version))
+    elif isinstance(version, str):
+        try:
+            if int(version) in iati.version.STANDARD_VERSIONS_MAJOR:
+                return version
+        except ValueError:
+            pass
+
+        version = iati.Version(version)
+
+    try:
+        if version in iati.version.STANDARD_VERSIONS:
```

Comment:
`resources` as a whole needs looking at. I'd suggest that changes to the module should be performed together, rather than making some smaller changes here, and having to modify them again before they hit `dev` or `master`.

Reply:
ok happy with this...